src/api/posts.js

```diff
@@ -0,0 +1,40 @@
+import { BASE_URL } from './api';
+
+const request = (url, options) => fetch(`${BASE_URL}${url}`, options)
+  .then((response) => {
+    if (!response.ok) {
+      throw new Error(`${response.status} - ${response.statusText}`);
+    }
+
+    return response.json();
+  })
+  .then(result => result.data);
+
+export const getUsers = () => request('/users');
```

Comment:
File is called `posts.js`.

Either rename the file and keep all api configurations here, or move users and comments controls to separate files.